bundles/specmate-emfrest/src/com/specmate/emfrest/authentication/UserSessionAndUser.java

```diff
@@ -0,0 +1,24 @@
+package com.specmate.emfrest.authentication;
+
+import com.specmate.usermodel.User;
+import com.specmate.usermodel.UserSession;
+
+public class UserSessionAndUser {

```

Comment:
Is this needed? Would Pair<UserSession,User> be enough? (e.g. from apache commons)